.github/workflows/publish-arc.yaml

```diff
@@ -71,22 +81,13 @@ jobs:
             echo "RELEASE_TAG_NAME=${{ github.event.inputs.release_tag_name }}" >> $GITHUB_ENV
           fi
 
-      - name: Resolve push to registries
```

Comment:
To be extra sure, we still need this step regardless of we set up the `VERSION` env for `make github-release` ☺️

Reply:
100% ! I renamed the variables to reduce redundancy

Reply:
Ah! It looks even more nicer now!